pkg/cmd/start/main.go

```diff
@@ -57,6 +57,7 @@ func NewStartCommand() *cobra.Command {
 	cmd.Flags().Int32("cr-metrics-port", 8686, "The metrics port for Operator and/or Custom Resource based metrics")
 	cmd.Flags().String("jaeger-agent-hostport", "localhost:6831", "The location for the Jaeger Agent")
 	cmd.Flags().Bool("tracing-enabled", false, "Whether the Operator should report its own spans to a Jaeger instance")
+	cmd.Flags().Bool("operand-metrics-enabled", false, "Whether the Operator should report metrics about the operands.")
```

Comment:
Why do we need this? The PR was so closed to get merged without this feature... Remember that smaller PRs get reviewed/merged faster.

Reply:
Well, I tried to maintain a similar option like one that we have for tracing.

Reply:
Also, I don't want to fetch the instances and do all the metrics logic if it's not requires, so people don't want to do it have this option to disable.

Reply:
It's not required, and it serves a different purpose as the tracing flag. The tracing flag serves as debugging help for the operator, while the metrics provide an insight into the setup over time.

In fact, I think we can even consider removing the tracing-enabled flag. Could you create a new issue to discuss that?

Reply:
Removed! I'll create the issue.